phpcs.xml.dist

```diff
@@ -16,6 +16,9 @@
 	<!-- Show progress. -->
 	<arg value="p"/>
 
+	<!-- Show rule name. -->
+	<arg value="s"/>
```

Comment:
FYI: This can be simplified by adding it to the previous directive, i.e.:
```xml
	<!-- Show progress and sniff errorcodes. -->
	<arg value="ps"/>
```

Reply:
Thanks. But there's no particular issue with them being separate? Looks clearer to me.

Reply:
Both should work AFAIK, so whatever you prefer, just wanted to let you know it was possible.